config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-periodics-janitor.yaml

```diff
@@ -24,4 +24,5 @@ periodics:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-digitalocean
       testgrid-tab-name: capdo-periodic-janitor
-      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
+      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-do-alerts@kubernetes.io
+      testgrid-num-failures-to-alert: "2"
```

Comment:
What's the significance of  2 failures needed before triggering an alert?

Reply:
because one can be just a flaky or other infra thing

Reply:
Ahh ok, thank you. The classic `1 is none, 2 is 1` . :smile: